vllm/core/scheduler.py

```diff
@@ -914,6 +992,10 @@ def _schedule_default(self) -> SchedulerOutputs:
                                                curr_loras,
                                                enable_chunking=False)
 
+        if len(prefills.seq_groups
+               ) == 0 and self.scheduler_config.policy == "sp":
```

Comment:
Shouldn't this be "priority"?

Reply:
Yes, changed to priority. The earlier use of "sp" was for strict priority. Based on the design discussion, the priority policy is general enough.